src/main/scala/com/lightbend/kafkalagexporter/AppConfig.scala

```diff
@@ -18,6 +18,11 @@ import scala.util.Try
 object AppConfig {
```

Comment:
Now that we have more than 1 reporter lets generate them based on configuration and only create them when necessary.  Example `application.conf`:

```
kafka-lag-exporter {
  reporters {
	prometheus {
      port = 8000
    }
    graphite {
      host = "foo"
      port = 9999
    }
  }
}
```